applying-stpa/l2_reqs.txt

```diff
@@ -0,0 +1,131 @@
+[('REQ-1', 'OpenAPS should be able to detect if dependencies are completely installed to match the current OpenAPS algorithm'),
```

Comment:
Should the L2 requirements also look like the L1?    Different syntax being used here - ie. "[('REQ-1', ..."

Reply:
This one's done

Reply:
It is automatically written to the file, L2 must have been done before that was implemented